src/storage/db_interface_frontend.py

```diff
@@ -329,9 +329,10 @@ def find_failed_analyses(self) -> Dict[str, List[str]]:
         return {entry['_id']: entry['UIDs'] for entry in query_result}
 
     def get_data_for_dependency_graph(self, uid):
-        data = list(self.file_objects.find({'parents': uid},
-                                           {'_id': 1, 'processed_analysis.elf_analysis': 1,
-                                            'processed_analysis.file_type': 1, 'file_name': 1}))
+        data = list(self.file_objects.find(
+            {'parents': uid}, 
```

Comment:
it seems the trailing space in this line displeased the CI:

> 00:25:54 =================================== FAILURES ===================================
00:25:54 _______________________ FLAKE8-check(ignoring E501 W503) _______________________
00:25:54 /home/jenkins/workspace/xenial/src/storage/db_interface_frontend.py:333:30: W291 trailing whitespace